meson.build

```diff
@@ -280,7 +280,11 @@ if build_standalone and get_option('plugin_uefi')
   efiboot = dependency('efiboot')
   objcopy = find_program ('objcopy')
   readelf = find_program ('readelf')
-  tpm2tss = dependency('tss2-esys', version : '>= 2.0')
+  tpm2tss = dependency('tss2-esys', version : '>= 2.0', required: false)
```

Comment:
I'm a little worried this could lead to an undefined situation.  Depending on what optimization meson does before evaluating `-Dplugin_tpm=true` and missing tsys could lead to a pass potentially.

So Perhaps lines 270-271 should instead show an error if it's missing?

Reply:
Maybe it might be cleaner if we did the `required: false` above both checks, and then handled the unfound case as a failure if the TPM plugin is enabled.

Reply:
I've pushed a new version that does this. Until tomorrow dude.